### PR TITLE
Convert to use Github actions

### PR DESCRIPTION
**Pull request by norkans7**

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -0,0 +1,64 @@
+name: CI
+on: [push, pull_request]
+jobs:
+  test:
+    name: Test
+    strategy:
+      matrix:
+        pg-version: ['9.6', '10', '11']
```
Comment (rowanseymour) [inline .github/workflows/ci.yml]: how come we're still doing 9.6 here?

File: backends/rapidpro/backend_test.go [hunk 2]
```diff
@@ -65,6 +65,13 @@ func (ts *BackendTestSuite) SetupSuite() {
 		log.Fatalf("unable to start backend for testing: %v", err)
 	}
 
+	// read our schema sql
+	sqlSchema, err := ioutil.ReadFile("schema.sql")
```
Comment (rowanseymour) [inline backends/rapidpro/backend_test.go]: FYI the postgres action actually has an option by where you provide a directory of SQL scripts to be run when database is being initialized.. but couldn't get it to work when I was converting mailroom..

File: .github/workflows/ci.yml (hunk 1 above)
Comment (norkans7) [reply .github/workflows/ci.yml]: Can remove that, I had assumed we still want to support that for the time being

File: backends/rapidpro/backend_test.go (hunk 2 above)
Comment (norkans7) [reply backends/rapidpro/backend_test.go]: I had the same issue as well

File: .github/workflows/ci.yml [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+    - name: Install PostgreSQL
+      uses: harmon758/postgresql-action@v1
+      with:
+        postgresql version: ${{ matrix.pg-version }}
+        postgresql db: courier_test
+        postgresql user: courier
+        postgresql password: courier
+
+    - name: Install Linux packages
+      run: sudo apt install -y --no-install-recommends postgresql-client
```
Comment (rowanseymour) [inline .github/workflows/ci.yml]: if you're not doing the sql loading in a step, then you don't need this

Comment (norkans7) [reply .github/workflows/ci.yml]: right

**Pull request merged**